package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "sealion",
-    "version": "0.1.0",
+    "version": "0.3.0",
```

Comment:
This change will constitute an update in content verbiage, without adding or removing a requirement, so we should bump this to `0.3.1`